### PR TITLE
Exams by Chernykova Daria ІМ-22

### PR DESCRIPTION
**Pull request by chErnykovaDD**
I've completed 2 tasks: 
1) find.js
2) count.js

### CODE REVIEW

File: Tasks/find.js [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 const find = (object, ...rest) => {
```
Comment (PavloVlasiuk) [inline Tasks/find.js:4]: I think, the name of the function is low understandable. Consider changing name, for examle: getKey or findKey.

**Review by PavloVlasiuk — COMMENTED** [review]
I have no idea what to change. Great job

**Review by PavloVlasiuk — APPROVED** [review]
It is easier to understand the meaning of function now. Well done!

**Review by PavloVlasiuk — APPROVED** [review]
I have no idea what to change. Great job

File: Tasks/count.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
-  {
-  }
-  keys = Object.keys(obj);
-  keys.forEach((key) => {
-    [];
+const sumOfNumbers = (obj) => {
+  let result = 0;
+
+  const keys = Object.keys(obj);
+
```
Comment (tshemsedinov) [inline Tasks/count.js:8]: Why do you have empty lines here?

File: Tasks/count.js [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
+  keys.map((key) => {
```
Comment (tshemsedinov) [inline Tasks/count.js:9]: Use `map` only to create outgoing array as a result of `map` call but not to change values in outer scope

File: Tasks/find.js [hunk 4]
```diff
@@ -1,17 +1,13 @@
...
-find = (object, ...rest) => {
-  value = rest.pop(1);
-  [];
-  for (name in object) {
-    if (object[name] !== value) {
-    } else {
-      if (typeof name) return name;
-      if (typeof object) return object;
-    }
+const getKey = (object, ...rest) => {
```
Comment (tshemsedinov) [inline Tasks/find.js:4]: We do not need rest operator here, just s single value

**Review by tshemsedinov — APPROVED** [review]

**Pull request closed without merging**